src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -6,7 +6,38 @@
      * Напишите метод toBinaryString(), который принимает на вход целое число value,
      * а возвращает String с представлением этого числа в двоичном виде.
      */
-    public String toBinaryString(int value) {
-        return null;
+    public static String toBinaryString(int value) {
+        StringBuilder str = new StringBuilder();
```

Comment:
This solution look like very complexity. try to simplify it) do we need method `sumOfNumber` here?